docs/reference/commands/certutil.asciidoc

```diff
@@ -73,15 +73,18 @@ directory name, you must also specify a file name in the `--name` command
 parameter or in the `filename` field in an input YAML file.
 
 You can optionally provide IP addresses or DNS names for each instance. If
-neither IP addresses nor DNS names are specified, the Elastic stack products
+neither IP addresses nor DNS names are specified, the Elastic Stack products
 cannot perform hostname verification and you might need to configure the
 `verification_mode` security setting to `certificate` only. For more information
 about this setting, see <<security-settings>>.
 
-All certificates that are generated by this command are signed by a CA. You can
-provide your own CA with the `--ca` or `--ca-cert` parameters. Otherwise, the
-command automatically generates a new CA for you. For more information about
-generating a CA, see the <<certutil-ca,CA mode of this command>>.
+All certificates that are generated by this command are signed by a CA unless
+the `--self-signed` parameter is specified. You can provide your own CA with the
+`--ca` or `--ca-cert` and `--ca-key` parameters. Otherwise, the command automatically generates a new CA for you.
+deprecated:[7.11.0,"Generating certificates without specifying a CA certificate and key is deprecated. In the next major version you must provide a CA certificate unless the `--self-signed` option is specified."]
```

Comment:
> deprecated:[7.11.0,"Generating certificates without specifying a CA certificate and key is deprecated. In the next major version you must provide a CA certificate unless the `--self-signed` option is specified."]

I've combined this information into a single paragraph so that it's (hopefully) clearer what's deprecated.